content/languages/c/criterion.md

```diff
@@ -30,6 +32,15 @@ Test(add_test, should_add_integers, .description = "should add integers") {
 }
 ```
 
+## Assertions
+
+_TBD_
+
+[Assertions reference](https://criterion.readthedocs.io/en/master/assert.html) in Criterion documentation.
```

Comment:
```suggestion
[Assertions reference](https://criterion.readthedocs.io/en/master/assert.html) can be found in the Criterion documentation.
```

Reply:
This file will have its own PR too, but I will merge this one since the file is already there, potentially known to the users.